src/documents/RemoteModelAPI.md

```diff
@@ -0,0 +1,364 @@
+                handlers will execute as part of this event sequence.
+            </td>
+        </tr>
+        <tr>
+            <th>Type</th>
+            <td>default</td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dt>data</dt>
+                <dd>
+                    The payload returned from the resolved <code>fetchImpl</code> promise.
+                </dd>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### onWrite
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                Fires an event sequence that executes before <code>writeImpl</code> is called.
+            </td>
+        </tr>
+        <tr>
+            <th>Type</th>
+            <td>default</td>
+        </tr>
+    </tbody>
+</table>
+
+### onWriteError
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                Fired if the <code>onWrite</code> sequence or <code>writeImpl</code> promises are rejected. By default
+                the <code>afterWrite.unblock</code> (unblocks the request queue) handler will execute.
+            </td>
+        </tr>
+        <tr>
+            <th>Type</th>
+            <td>default</td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                Any parameters passed along to the rejected promise. This may be an <code>Error</code> object, an error
+                message, and/or some other means of indicating the rejection reason.
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### afterWrite
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                Fires an event sequence that executes after <code>writeImpl</code> is called. By default the
+                <code>afterWrite.unblock</code> (unblocks the request queue) handler will execute as part of this event
+                sequence.
+            </td>
+        </tr>
+        <tr>
+            <th>Type</th>
+            <td>default</td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dt>data</dt>
+                <dd>
+                    The payload returned from the resolved <code>writeImpl</code> promise.
+                </dd>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Model
+
+<table>
+    <thead>
+        <tr>
+            <th>Model Path</th>
+            <th>Description</th>
+            <th>Default</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>local</code></td>
+            <td>
+                Provides the buffered mapping of the portions of the components's model to the remote data that should
+                be kept in sync. An implementor must setup the <a href="ModelRelay.md">model relay(s)</a> between the
+                component's model data and what is contained in the <code>local</code> path.
```

Comment:
Needn't be in the component itself - better would be "between the provider of the model data and this component's `local" path